src/node_options.cc

```diff
@@ -113,6 +113,13 @@ void EnvironmentOptions::CheckOptions(std::vector<std::string>* errors) {
     errors->push_back("invalid value for --http-parser");
   }
 
+  if (!unhandled_rejections.empty() &&
```

Comment:
(if you add the `= "warn"` above, you can drop the first part of the conditional)

Reply:
The default behavior is actually not absolutely indentical to `warn` as I outlined in the description. If the flag is used, the deprecation warning will not be printed anymore. That is why I implemented it that way.